lib/do_volume/do_create.js

```diff
@@ -0,0 +1,208 @@
+                            if (net) {
+                                ctx.networks.push(net);
+                            }
+
+                            nextNet(getNetErr);
+                        });
+                }
+            }, next);
+        },
+        function createVolume(ctx, next) {
+            var createVolumeParams = {
+                type: 'tritonnfs',
+                name: opts.name,
+                networks: ctx.networks,
+                size: ctx.size
+            };
+
+            if (opts.type) {
+                createVolumeParams.type = opts.type;
+            }
+
+            self.top.tritonapi.cloudapi.createVolume(createVolumeParams,
+                function onRes(volCreateErr, volume) {
+                    ctx.volume = volume;
+                    next(volCreateErr);
+                });
+        },
+        function maybeWait(ctx, next) {
+            var distraction;
+            var waitTimeout = opts.wait_timeout === undefined ?
+                    undefined : opts.wait_timeout * 1000;
+
+            if (!opts.wait) {
+                next();
+                return;
+            }
+
+            if (process.stderr.isTTY && opts.wait.length > 1) {
+                distraction = distractions.createDistraction(opts.wait.length);
+            }
+
+            self.top.tritonapi.cloudapi.waitForVolumeStates({
+                id: ctx.volume.id,
+                states: ['ready', 'failed'],
+                timeout: waitTimeout
+            }, function onWaitDone(waitErr, volume) {
+                if (distraction) {
+                    distraction.destroy();
+                }
+
+                ctx.volume = volume;
+                next(waitErr);
+            });
+        },
+        function outputRes(ctx, next) {
+            assert.object(ctx.volume, 'ctx.volume');
+
+            if (opts.json) {
+                console.log(JSON.stringify(ctx.volume));
+            } else {
+                console.log(JSON.stringify(ctx.volume, null, 4));
+            }
+        }
+    ]}, cb);
+}
+
+do_create.options = [
+    {
+        names: ['help', 'h'],
+        type: 'bool',
+        help: 'Show this help.'
+    },
+    {
+        group: 'Create options'
+    },
+    {
+        names: ['name', 'n'],
+        helpArg: 'NAME',
+        type: 'string',
+        help: 'Volume name. If not given, one will be generated server-side.'
+    },
+    {
+        names: ['type', 't'],
+        helpArg: 'TYPE',
+        type: 'string',
+        help: 'Volume type. Default is "tritonnfs".'
+    },
+    {
+        names: ['size', 's'],
+        type: 'string',
+        helpArg: 'SIZE',
+        help: 'The `size` input parameter must match the following regular ' +
+            'expression: /(\d+)(g|m|G|M|gb|mb|GB|MB)/ All units are in ' +
+            'ibibytes (mebibytes and gibibytes). `g`, `G`, `gb` and `GB` ' +
+            'stand for "gibibytes". `m`, `M`, `mb` and `MB` stand for ' +
+            '"mebibytes".',
```

Comment:
> This help string could be tighter I think. 

Sounds good, will fix.

> I'm also biasing to not supporting multiple spellings of the units.

I agree, although FWIW it departs from what [RFD 26 states](https://github.com/joyent/rfd/blob/master/rfd/0026/README.md#size):

```
When specifying a volume size without a suffix, gibibytes are assumed: a size of 10 means 10 gibibytes. When using suffixes in the size parameter, binary units are assumed. That is: 10g means 10 Gibibtyes, same for 10G, 10gb and 10GB.
```